### ISSUE TITLE
Accessibility concerns 

### ISSUE THREAD

**Issue by DavidKozdra** [riccardobertolini/lofi-music#60]
There are a few accessibility issues that should be fixed 

Light house requires 
The buttons need a name  forms need a label 

Properly ordered headings that do not skip levels convey the semantic structure of the page, making it easier to navigate and understand when using assistive technologies. [Learn more about heading order](https://dequeuniversity.com/rules/axe/4.7/heading-order).

Needed items 

The play buttons are not currently selectable need a tab index attribute 

The modal that pops up for settings could request conditionally all other tab indexes to be -1 when open to make it easier to close 

The modal close button needs a :focus-visible selector and to close on key press

**Comment by DavidKozdra** [comment]
forgot the date display for calendar should have 
cursor : pointer and should seem selectable in some way

**Comment by DavidKozdra** [comment]
Made a branch almost done will clean this up by tomorrow

**Comment by DavidKozdra** [comment]
Most if not all of this is solved in the last PR
